Fix this silly warning about superfluous parentheses

```
src/run.cpp:104:17: warning: equality comparison with extraneous parentheses [-Wparentheses-equality]
  if ((mesh.rank==0)){
       ~~~~~~~~~^~~
```

--- a/src/run.cpp
+++ b/src/run.cpp
@@ -96,7 +96,7 @@ void hipBone_t::Run(){
   size_t NflopsNekbone =   (15*Np  //CG flops
                           + 19*Np+12*Nq*Nq*Nq*Nq )*mesh.NelementsGlobal*Niter; //flops per CG iteration
 
-  if ((mesh.rank==0)){
+  if (mesh.rank==0){
     printf("hipBone: %d, " hlongFormat ", %4.4f, %d, %1.2e, %4.1f, %4.1f, %1.2e; N, DOFs, elapsed, iterations, time per DOF, avg BW (GB/s), avg GFLOPs, DOFs*iterations/ranks*time \n",
            mesh.N,
            Ndofs,

